cmd/workspace/model-registry/model-registry.go

```diff
@@ -2136,7 +2136,7 @@ func newTestRegistryWebhook() *cobra.Command {
 	// TODO: short flags
 	cmd.Flags().Var(&testRegistryWebhookJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().Var(&testRegistryWebhookReq.Event, "event", `If event is specified, the test trigger uses the specified event.`)
+	cmd.Flags().Var(&testRegistryWebhookReq.Event, "event", `If event is specified, the test trigger uses the specified event. Supported values: [COMMENT_CREATED, MODEL_VERSION_CREATED, MODEL_VERSION_TAG_SET, MODEL_VERSION_TRANSITIONED_STAGE, MODEL_VERSION_TRANSITIONED_TO_ARCHIVED, MODEL_VERSION_TRANSITIONED_TO_PRODUCTION, MODEL_VERSION_TRANSITIONED_TO_STAGING, REGISTERED_MODEL_CREATED, TRANSITION_REQUEST_CREATED, TRANSITION_REQUEST_TO_ARCHIVED_CREATED, TRANSITION_REQUEST_TO_PRODUCTION_CREATED, TRANSITION_REQUEST_TO_STAGING_CREATED]`)
```

Comment:
These lines become huge; is there a better way to format this?

Reply:
done, I added new line formatting for every enum value. It works but it does use up some vertical space in the help doc outputs:
```
Flags:
      --event RegistryWebhookEvent   If event is specified, the test trigger uses the specified event. Supported values: [
                                       COMMENT_CREATED,
                                       MODEL_VERSION_CREATED,
                                       MODEL_VERSION_TAG_SET,
                                       MODEL_VERSION_TRANSITIONED_STAGE,
                                       MODEL_VERSION_TRANSITIONED_TO_ARCHIVED,
                                       MODEL_VERSION_TRANSITIONED_TO_PRODUCTION,
                                       MODEL_VERSION_TRANSITIONED_TO_STAGING,
                                       REGISTERED_MODEL_CREATED,
                                       TRANSITION_REQUEST_CREATED,
                                       TRANSITION_REQUEST_TO_ARCHIVED_CREATED,
                                       TRANSITION_REQUEST_TO_PRODUCTION_CREATED,
                                       TRANSITION_REQUEST_TO_STAGING_CREATED,
                                     ]
```